coderd/metricscache/metricscache_test.go

```diff
@@ -67,38 +72,47 @@ func TestCache_TemplateWorkspaceOwners(t *testing.T) {
 		Provisioner:    database.ProvisionerTypeEcho,
 		CreatedBy:      user1.ID,
 	})
-	require.Eventuallyf(t, func() bool {
-		count, ok := cache.TemplateWorkspaceOwners(template.ID)
-		return ok && count == 0
-	}, testutil.WaitShort, testutil.IntervalMedium,
-		"TemplateWorkspaceOwners never populated 0 owners",
-	)
+
+	// Wait for both ticker functions to be created (template build times and deployment stats)
+	trapTickerFunc.MustWait(ctx).MustRelease(ctx)
+	trapTickerFunc.MustWait(ctx).MustRelease(ctx)
+
+	_, wait := clock.AdvanceNext()
+	wait.MustWait(ctx)
+	_, wait = clock.AdvanceNext()
+	wait.MustWait(ctx)
```

Comment:
You only need to trigger one tick in each of these. Both the template build times and deployment stats have the same interval, so both get triggered on each tick.

Also, I think it's more understandable and concise to write:

```
clock.Advance(time.Minute).MustWait(ctx)
```